core/chains/evm/log/broadcaster.go

```diff
@@ -614,7 +615,7 @@ func (b *broadcaster) Resume() {
 
 // test only
 func (b *broadcaster) LogsFromBlock(bh common.Hash) int {
-	return len(b.logPool.logsByBlockHash[bh])
+	return b.logPool.testOnly_getNumLogsForBlock(bh)
```

Comment:
Please can we move this to a `exports_test.go` or `helpers_test.go` instead. There shouldn't be production code anywhere that is "test only"

Reply:
This whole function(LogsFromBlock) is test only. It was already there before, and moving it to helpers_test.go will be lot of refactoring for broadcaster.go file, which I want to avoid in this PR, where the goal is only to add tests for logPool.

I could do this as part of another PR which writes unit-tests for broadcaster.go file.